doc/BUILD_ON_DOCKER.md

```diff
@@ -0,0 +1,40 @@
+# Steps:
+
+## Step 1: clone the repository
+
+```{bash}
+git clone https://github.com/AppFlowy-IO/appflowy.git
+```
+
+## Step 2: cd into docker-buildfiles
+
+```{bash}
+cd ./appflowy/doc/docker-buildfiles
+```
+
+## Step 3: Provide access of appflowy to X session
+
+```{bash}
+xhost local:root
+```
+
+## Step 4: build the docker image
+
+```{bash}
+docker-compose build
```

Comment:
I was required to put a "sudo" in front of this... Normal?

Reply:
fixed in readme, you need to add docker to the user group